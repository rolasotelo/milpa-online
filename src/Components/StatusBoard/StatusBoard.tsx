import React from "react";
import useGameContext from "../../Hooks/useGameContext/useGameContext";

interface Props {}

const StatusBoard = (props: Props) => {
  const context = useGameContext();
  const card = context.selectedCard.card;
  return (
    <div className="h-24 bg-mexicanBoneLight rounded-lg mt-1 mx-1 px-2">
      {card ? <p>{`${card.name}: ${card.rules}`}</p> : null}
    </div>
  );
};

export default StatusBoard;
