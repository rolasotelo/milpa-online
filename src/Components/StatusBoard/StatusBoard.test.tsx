import React from "react";
import renderer from "react-test-renderer";

import StatusBoard from "./StatusBoard";

describe("<StatusBoard />", () => {
  const defaultProps = {};
  const wrapper = renderer.create(<StatusBoard {...defaultProps} />);

  test("render", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
