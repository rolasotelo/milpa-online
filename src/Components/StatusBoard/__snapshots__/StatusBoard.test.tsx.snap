// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StatusBoard /> render 1`] = `
<div
  className="h-16 bg-mexicanBoneLight"
>
  Your Turn
</div>
`;
